lms/static/scripts/frontend_apps/components/FilePickerApp.js

```diff
@@ -94,13 +98,37 @@ export default function FilePickerApp({ onSubmit }) {
   // Submit the form after a selection is made via one of the available
   // methods.
   useEffect(() => {
+    async function createAssignment() {
+      const data = { ...createAssignmentAPI.data };
+      data.content = content;
+      data.groupset = groupConfig.groupSet;
+      const assignment = await apiCall({
+        authToken,
+        path: createAssignmentAPI.path,
+        data: data,
+      });
+      setExtLTIAssignmentId(assignment.ext_lti_assignment_id);
+    }
+
+    if (content && createAssignmentAPI && !extLTIAssignmentId) {
```

Comment:
Ony call the API if:
- Content already selected
- API enabled the config
- And we don't have already called it